### PR TITLE
ISPN-12311 Fix data access % in chart

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-12311
https://issues.redhat.com/browse/JDG-3922

### CODE REVIEW

File: src/app/Caches/CacheMetrics.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ const CacheMetrics = (props: {
                 {x: 'Removes Misses', y: stats.remove_misses},
                 {x: 'Evictions', y: stats.evictions}
               ]}
-              labels={({datum}) => `${datum.x}: ${datum.y}%`}
+              labels={({datum}) => `${datum.x}:${displayUtils.formatNumber((datum.y * 100)/all)}%`}
```
Comment (pruivo) [inline src/app/Caches/CacheMetrics.tsx:139]: shouldn't check if `all != 0`?

File: src/app/GlobalStats/GlobalStats.tsx [hunk 2]
```diff
@@ -203,7 +203,7 @@ const GlobalStats: React.FunctionComponent<any> = props => {
                     { x: 'Removes Misses', y: stats.remove_misses },
                     { x: 'Evictions', y: stats.evictions }
                   ]}
-                  labels={({ datum }) => `${datum.x}: ${datum.y}%`}
+                  labels={({ datum }) => `${datum.x}: ${displayUtils.formatNumber((datum.y * 100)/allOps())}%`}
```
Comment (pruivo) [inline src/app/GlobalStats/GlobalStats.tsx:206]: shouldn't check if `allOps() != 0`?

Comment (karesti) [reply src/app/GlobalStats/GlobalStats.tsx:206]: no, not needed

File: src/app/Caches/CacheMetrics.tsx (hunk 1 above)
Comment (karesti) [reply src/app/Caches/CacheMetrics.tsx:139]: not needed

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**